### PR TITLE
Geo dcat charter

### PR DESCRIPTION
**Pull request by rob-metalinkage**
PR to merge GeoDCAT charter branch into master - can be actioned after review period

### CODE REVIEW

**Comment by rob-metalinkage** [comment]
Note - added a github action that can be manually triggered to update generated docs.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 This is the open repository for work done within the OGC GeoSemantics DWG. 
 
+# GeoSPARQL
```
Comment (TallTed) [inline README.md]:
```suggestion
## GeoSPARQL
```

File: README.md [hunk 2]
```diff
@@ -8,3 +10,8 @@ The draft charter: [OGC GeoSPARQL SWG Charter](https://github.com/opengeospatial
 
 The white paper: [OGC Benefits of Representing Spatial Data Using Semantic and Graph Technologies](https://github.com/opengeospatial/geosemantics-dwg/blob/master/white_paper/wp.pdf)
 
+# GeoDCAT
```
Comment (TallTed) [inline README.md]:
```suggestion
## GeoDCAT
```

File: README.md (hunk 1 above)
Comment (TallTed) [reply README.md]: You might add a proper `H1` or `#` line with a title for the document, above the current line 1 ... but there should only ever be one such `H1` or `#` line in a document; there may be as many `H2` or `##` lines as there are sections.

File: README.md [hunk 3]
```diff
@@ -8,3 +10,8 @@ The draft charter: [OGC GeoSPARQL SWG Charter](https://github.com/opengeospatial
...
+
+A joint SWG (with the MetaCat DWG) for the publication of GeoDCAT, GeoDCAT-AP and other profiles of DCAT to support use of OGC standards in.
```
Comment (TallTed) [inline README.md]:
```suggestion
A joint SWG (with the MetaCat DWG) for the publication of GeoDCAT, GeoDCAT-AP, and other profiles of DCAT to support use of OGC standards.
```

File: geodcat_swg_charter/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
+# GeoDCAT SWG Charter
+
+- draft to be released for DWG review.
+
+A joint SWG (with the MetaCat DWG) for the publication of GeoDCAT, GeoDCAT-AP and other profiles of DCAT to support use of OGC standards in.
```
Comment (TallTed) [inline geodcat_swg_charter/README.md]:
```suggestion
A joint SWG (with the MetaCat DWG) for the publication of GeoDCAT, GeoDCAT-AP, and other profiles of DCAT to support use of OGC standards.
```

File: geodcat_swg_charter/clause_1.adoc [hunk 5]
```diff
@@ -0,0 +1,10 @@
+== Purpose of the Standards Working Group
+
+////
+Proposers will describe the purpose of the Standards Working Group and its overall mission in relation to OGC processes, the OGC standards baseline, and OGC’s business plan.
+////
+
+The purpose of this SWG is to revise, publish and maintain GeoDCAT, a spatio-temporal profile of the W3C DCAT Recommendation <<DCAT>> , and provide guidance about its use and further specialisation.
```
Comment (TallTed) [inline geodcat_swg_charter/clause_1.adoc]:
```suggestion
The purpose of this SWG is to revise, publish, and maintain GeoDCAT, a spatio-temporal profile of the W3C DCAT Recommendation <<DCAT>> , and provide guidance about its use and further specialisation.
```

File: geodcat_swg_charter/clause_3.adoc [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+- prepare and validate illustrative examples using the machine-readable description
+- define a mechanism for describing specialised profiles of GeoDCAT
+- publish using current OGC specification document methodology
+
+Future activities will include definition of additional specialised profiles as required and requested by specific applications domains (via OGC Domain Working Group inputs).
+
+
+=== Statement of relationship of planned work to the current OGC standards baseline
+
+This scope is primarily aimed to progress the concept of GeoDCAT from an informative discussion paper into a normative and supported resource.
```
Comment (TallTed) [inline geodcat_swg_charter/clause_3.adoc]:
```suggestion
This scope is primarily intended to progress the concept of GeoDCAT from an informative discussion paper into a normative and supported resource.
```

File: geodcat_swg_charter/clause_3.adoc [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+Further specialisations will provide transparent, interoperable and validatable patterns for the inclusion of OGC standards conformant resources into DCAT based catalogue implementations.
```
Comment (TallTed) [inline geodcat_swg_charter/clause_3.adoc]:
```suggestion
Further specialisations will provide transparent, interoperable, and validatable patterns for the inclusion of OGC standards conformant resources into DCAT based catalogue implementations.
```

File: geodcat_swg_charter/clause_3.adoc [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+=== Is This a Persistent SWG
+
+[x] YES
+
+[ ] NO
+
+=== When can the SWG be Inactivated
+
+The SWG can be inactivated once a registration mechanism (policies, technical standards and infrastructure support) for further application domain specialised profiles of DCAT allows use of the GeoDCAT profile without the need for additional modification.
```
Comment (TallTed) [inline geodcat_swg_charter/clause_3.adoc]:
```suggestion
The SWG can be inactivated once a registration mechanism (policies, technical standards, and infrastructure support) for further application domain specialised profiles of DCAT allows use of the GeoDCAT profile without the need for additional modification.
```

File: geodcat_swg_charter/clause_7.adoc [hunk 9]
```diff
@@ -0,0 +1,6 @@
+== Domain Working Group Endorsement
+
+This SWG scope addresses requirements for interoperable best practices in both geosemantics and metadata catalogs, and hence will report to both the Geosemantics and MetCat Domain Working Groups.
+
+The Chairs of the Geosemantics DWG, Joseph Abhayaratna and Linda van den Brink, and the chairs of the MetaCat DWG, Danny Vandenbrouke and Byron Cochrane do formally endorse the formation of the GeoDCAT Standards Working Group (SWG).
```
Comment (TallTed) [inline geodcat_swg_charter/clause_7.adoc]:
```suggestion
The Chairs of the Geosemantics DWG, Joseph Abhayaratna and Linda van den Brink, and the chairs of the MetaCat DWG, Danny Vandenbrouke and Byron Cochrane, do formally endorse the formation of the GeoDCAT Standards Working Group (SWG).
```

File: geodcat_swg_charter/clause_8.adoc [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+// Review and update normative references
+
+==== DCAT-AP
+
+
+==== W3C Linked Building Data Community Group
+
+===== Scope of Activity
+
+The European Union publishes <<DCAT-AP>> - a profile of DCAT to meet administrative requirements for data publication in EU context.
```
Comment (TallTed) [inline geodcat_swg_charter/clause_8.adoc]:
```suggestion
The European Union publishes <<DCAT-AP>> — a profile of DCAT to meet administrative requirements for data publication in EU context.
```

File: geodcat_swg_charter/clause_9.adoc [hunk 11]
```diff
@@ -0,0 +1,25 @@
+== References
+
+[[DCAT]]
+[DCAT] Web: W3C: Data Catalog Vocabulary (DCAT) - Version 2, https://www.w3.org/TR/vocab-dcat-2/ (2020)
```
Comment (TallTed) [inline geodcat_swg_charter/clause_9.adoc]:
```suggestion
[DCAT] Web: W3C: Data Catalog Vocabulary (DCAT) — Version 2, https://www.w3.org/TR/vocab-dcat-2/ (2020)
```

File: geodcat_swg_charter/clause_9.adoc [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+
+[[DCAT3]]
+[DCAT3] Web: W3C: Data Catalog Vocabulary (DCAT) - Version 3, https://www.w3.org/TR/vocab-dcat-3/
```
Comment (TallTed) [inline geodcat_swg_charter/clause_9.adoc]:
```suggestion
[DCAT3] Web: W3C: Data Catalog Vocabulary (DCAT) — Version 3, https://www.w3.org/TR/vocab-dcat-3/
```

File: geodcat_swg_charter/clause_9.adoc [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+
+[[SDWBP]]
+[SDWBP]  Web: OGC & W3C: Spatial Data on the Web Best Practices, https://www.w3.org/TR/sdw-bp/
+
+[[GeoDCAT-AP]]
+[GeoDCAT-AP] Web: OGC : GeoDCAT-AP Discussion Paper, http://www.opengis.net/doc/dp/GeoDCAT-AP
+
+[[GeoDCAT-APv2]]
+[GeoDCAT-APv2] GeoDCAT-AP - Version 2.0.0 : SEMIC Editor's Draft 23 December 2020, https://semiceu.github.io/GeoDCAT-AP/drafts/latest/
```
Comment (TallTed) [inline geodcat_swg_charter/clause_9.adoc]:
```suggestion
[GeoDCAT-APv2] GeoDCAT-AP — Version 2.0.0 : SEMIC Editor's Draft 23 December 2020, https://semiceu.github.io/GeoDCAT-AP/drafts/latest/
```

File: geodcat_swg_charter/clause_8.adoc [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+
+This application profile is a specification for metadata records to meet the specific application needs of data portals in Europe while providing semantic interoperability with other applications on the basis of reuse of established controlled vocabularies (e.g. EuroVoc) and mappings to existing metadata vocabularies (e.g. Dublin Core, SDMX, INSPIRE metadata, etc.).
```
Comment (TallTed) [inline geodcat_swg_charter/clause_8.adoc]:
```suggestion
This application profile is a specification for metadata records to meet the specific application needs of data portals in Europe while providing semantic interoperability with other applications on the basis of reuse of established controlled vocabularies (e.g., EuroVoc) and mappings to existing metadata vocabularies (e.g., Dublin Core, SDMX, INSPIRE metadata).
```

File: geodcat_swg_charter/clause_8.adoc [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+
+The first meeting of the SWG will be held by telephone conference call at 6AM EDT on the first Thursday immediately following the week of Technical Committee approval of this Charter. Call-in information will be provided to the SWG's e-mail list and on the portal calendar in advance of the meeting.
+
+=== Projected on-going meeting schedule
+
+The work of the SWG will be carried out primarily by email and conference calls, possibly every two weeks, with face-to-face meetings perhaps at each of the OGC TC meetings.
+
+=== Supporters of this Charter
+
+The following people support this proposal and are committed to the Charter and projected meeting schedule. These members are known as SWG Founding or Charter members. The charter members agree to the SoW and IPR terms as defined in this charter. The charter members have voting rights beginning the day the SWG is officially formed. Charter Members are shown on the public SWG page.
```
Comment (TallTed) [inline geodcat_swg_charter/clause_8.adoc]:
```suggestion
The following people support this proposal and are committed to the Charter and projected meeting schedule. These members are known as SWG Founding or Charter Members. The Charter Members agree to the SoW and IPR terms as defined in this charter. The Charter Members have voting rights beginning the day the SWG is officially formed. Charter Members are shown on the public SWG page.
```

**Review by lvdbrink — APPROVED** [review]
I think this charter is good to go!

File: README.md [hunk 16]
```diff
@@ -12,6 +12,6 @@ The white paper: [OGC Benefits of Representing Spatial Data Using Semantic and G
 
 # GeoDCAT
 
-A joint SWG (with the MetaCat DWG) for the publication of GeoDCAT, GeoDCAT-AP and other profiles of DCAT to support use of OGC standards in. 
+A SWG, jointly hosted by the GeoSemantics DWG and the MetaCat DWG, for the publication of GeoDCAT, GeoDCAT-AP and other profiles of DCAT to support use of OGC standards in.
```
Comment (TallTed) [inline README.md]:
```suggestion
An SWG, jointly hosted by the GeoSemantics DWG and the MetaCat DWG, for the publication of GeoDCAT, GeoDCAT-AP, and other profiles of DCAT to support use of OGC standards. 
```

**Pull request merged**